e2etests/tests/evpn_routes.go

```diff
@@ -139,7 +139,9 @@ var _ = Describe("Routes between bgp and the fabric", Ordered, func() {
 				Eventually(func() error {
 					for exec := range routers.GetExecutors() {
 						evpn, err := frr.EVPNInfo(exec)
-						Expect(err).NotTo(HaveOccurred())
+						if err != nil {
```

Comment:
makes total sense, I wonder if you saw evidence in some failures as we (in theory) should be covered by the healthprobe that ensures the router pod is ready when the daemons are up

Reply:
ack, I think it was deliberate as in "this should never fail", but won't retrying won't hurt.